src/appleseedmaya/extensionattributes.cpp

```diff
@@ -235,6 +238,8 @@ namespace
             MFnNumericData::kFloat,
             0.0f,
             status);
+        numAttrFn.setSoftMin(0.0f);
```

Comment:
Shouldn't we allow negative values here? Just a question.

Reply:
We do, the moment a user enters a value outside the soft min/max, the float slider is rescaled to the new range. In Maya the softmin/max are just the initial slider range defaults.

Reply:
Sorry, I wasn't clear: what I mean is that the default slider range should reflect what's possible, with reasonable bounds. So IMHO it should offer to go from say -8 to +8, or even a tighter range, but it should cover both over- and under-exposition.

Reply:
Got it. Done.